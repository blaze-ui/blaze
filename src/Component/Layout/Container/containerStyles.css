.sm {
  background-color: #ECF0F1;
  height: 50vh;
}

.md {
  background-color: #ECF0F1;
  height: 75vh;
}

.xl {
  background-color: #ECF0F1;
  height: 100vh;
}
/*# sourceMappingURL=containerStyles.css.map */