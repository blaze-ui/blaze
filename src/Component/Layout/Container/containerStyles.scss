@import "./../../../Styles/globals.scss";

.sm{
    background-color: $containerFluid;
    height: 50vh;
}
.md{
    background-color: $containerFluid;
    height: 75vh;
}
.xl{
    background-color: $containerFluid;
    height: 100vh;
}