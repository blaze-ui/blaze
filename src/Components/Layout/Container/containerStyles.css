.sm {
  background-color: #ecf0f1;
  height: 20vh;
}

.md {
  background-color: #d0ace4;
  height: 60vh;
}

.xl {
  background-color: #aadb1e;
  height: 100vh;
}
/*# sourceMappingURL=containerStyles.css.map */