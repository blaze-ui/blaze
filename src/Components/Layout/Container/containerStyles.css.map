{
    "version": 3,
    "mappings": "AAEA,AAAA,GAAG,CAAA;EACC,gBAAgB,ECMD,OAAO;EDLtB,MAAM,EAAC,IAAI;CACd;;AACD,AAAA,GAAG,CAAA;EACC,gBAAgB,ECGD,OAAO;EDFtB,MAAM,EAAC,IAAI;CACd;;AACD,AAAA,GAAG,CAAA;EACC,gBAAgB,ECAD,OAAO;EDCtB,MAAM,EAAE,KAAK;CAChB",
    "sources": [
        "containerStyles.scss",
        "../../../Styles/globals.scss"
    ],
    "names": [],
    "file": "containerStyles.css"
}