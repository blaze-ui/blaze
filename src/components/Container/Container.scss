@import "../../styles/ColorSchema.scss";

.sm{
    background-color: $blue-grad;
    height:20vh;
}
.md{
    background-color: $dark-gray-grad;
    height:60vh;
}
.xl{
    background-color: $dark-gray-grad;
    height: 100vh;
}