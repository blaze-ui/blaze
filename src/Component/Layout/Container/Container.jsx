import React, { Component } from 'react';
import './containerStyles.scss';
import Proptypes from "prop-types";

export default class Container extends Component {
    render() {
      let typeName=this.props.type;
      let maxWidth=this.props.maxWidth;
      
      return (
        <div>
         {typeName ===  "fluid"       
            ? <div className={maxWidth}/>
            : <div style={{backgroundColor: '#F9690E',
                            height: '100vh'}}/>
          }

        </div>  
      );
    }
  }

Container.propType = {
    typeName: Proptypes.string,
    maxWidth: Proptypes.string
  };
  

