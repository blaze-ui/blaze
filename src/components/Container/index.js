/* eslint-disable jsx-a11y/label-has-associated-control */
/* eslint-disable jsx-a11y/label-has-for */
/* eslint-disable linebreak-style */
/* eslint-disable react/destructuring-assignment */
/* eslint-disable react/button-has-type */
/* eslint-disable react/prefer-stateless-function */
import React from "react";
import Proptypes from "prop-types";

import "./Container.css";

function Container(props) {
  const { maxWidth, typeName } = props;

  return (
    <div>
      {typeName === "fluid" ? (
        <div className={maxWidth} />
      ) : (
        <div style={{ backgroundColor: "#F9690E", height: "100vh" }} />
      )}
    </div>
  );
}

Container.propType = {
  typeName: Proptypes.string,
  maxWidth: Proptypes.string
};

export default Container;
