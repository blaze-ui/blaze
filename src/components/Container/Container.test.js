import React from "react";
import renderer from "react-test-renderer";
import Container from "./index";

test("Container tests", () => {
  const component = renderer.create(<Container> </Container>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
