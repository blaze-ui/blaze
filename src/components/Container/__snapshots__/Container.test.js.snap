// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container tests 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "#F9690E",
        "height": "100vh",
      }
    }
  />
</div>
`;
